processor/resourcedetectionprocessor/README.md

```diff
@@ -426,6 +426,27 @@ processors:
     override: false
 ```
 
+### Kubeadm Metadata
+
+Queries the K8S API server to retrieve the following resource attributes:
+
+    * k8s.cluster.name
+
+The following permissions are required:
+```yaml
+kind: ClusterRole
```

Comment:
Could that just be a `Role` instead with a fixed name (`kubeadm-config`) with only `get` permissions? I would expect that the name is fixed.

i.e.:

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: Role
metadata:
  name: some
rules:
  - apiGroups: [""]
    resources:
      - configmaps
    resourceNames:
      - kubeadm-config
    verbs: ["get"]
```